week09_compression/README.md

```diff
@@ -0,0 +1,54 @@
+# Week 9: Efficient model inference
+
+* Lecture: [slides](./lecture_compressed.pdf)
+* Seminar: [notebook](./practice.ipynb)
+* Homework: see [homework/README.md](homework/README.md)
+
+### Setup for the seminar notebook
+You can use [conda](https://docs.anaconda.com/free/miniconda/), [mamba](https://mamba.readthedocs.io/en/latest/user_guide/mamba.html) or [micromamba](https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html) to create the environment.
+
+```
+conda create -n inference python=3.10 pytorch torchvision torchaudio pytorch-cuda=11.8  matplotlib seaborn numpy tqdm ipywidgets jupyterlab_widgets jupyterlab tqdm -c pytorch -c nvidia -y
```

Comment:
Nit: it's usually best to pin package dependencies to avoid sudden API changes in the future